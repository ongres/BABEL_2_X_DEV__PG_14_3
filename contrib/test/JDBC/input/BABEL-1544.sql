-- Test LEN() function with BINARY and VARBINARY

-- VARBINARY INPUTS
declare @vb varbinary(10) = NULL
select len(@vb)
go

declare @vb varbinary(5)
set @vb = 0x90a;
select len(@vb)
go

declare @vb varbinary(1)
set @vb = 0x90a;
select len(@vb)
go

declare @vb varbinary(10)
set @vb = 0x90a;
select len(@vb)
go

declare @vb varbinary(7)
set @vb = 0x010a1a1a1a;
select len(@vb)
go

declare @vb varbinary(10)
set @vb = 0x0102030405060708090a021321321a321a3a213a21a;
select len(@vb)
go

declare @vb varbinary(10)
set @vb = 0x0102030405060708090a021a31321a321a321a321a321a3a213a21a;
select len(@vb)
go


declare @vb varbinary(10) = 0x0102030405060708090a021a31321a321a321a321a321a3a213a21a321a321a321a36513a216a03
select len(@vb)
go
