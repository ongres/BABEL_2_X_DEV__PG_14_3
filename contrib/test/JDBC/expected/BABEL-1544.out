
-- Test LEN() function with BINARY and VARBINARY
-- VARBINARY INPUTS
declare @vb varbinary(10) = NULL
select len(@vb)
go
~~START~~
int
<NULL>
~~END~~


declare @vb varbinary(5)
set @vb = 0x90a;
select len(@vb)
go
~~START~~
int
2
~~END~~


declare @vb varbinary(1)
set @vb = 0x90a;
select len(@vb)
go
~~START~~
int
1
~~END~~


declare @vb varbinary(10)
set @vb = 0x90a;
select len(@vb)
go
~~START~~
int
2
~~END~~


declare @vb varbinary(7)
set @vb = 0x010a1a1a1a;
select len(@vb)
go
~~START~~
int
5
~~END~~


declare @vb varbinary(10)
set @vb = 0x0102030405060708090a021321321a321a3a213a21a;
select len(@vb)
go
~~START~~
int
10
~~END~~


declare @vb varbinary(10)
set @vb = 0x0102030405060708090a021a31321a321a321a321a321a3a213a21a;
select len(@vb)
go
~~START~~
int
10
~~END~~



declare @vb varbinary(10) = 0x0102030405060708090a021a31321a321a321a321a321a3a213a21a321a321a321a36513a216a03
select len(@vb)
go
~~START~~
int
10
~~END~~

